pom.xml

```diff
@@ -81,6 +80,17 @@
             <artifactId>xmlunit-core</artifactId>
             <version>2.10.0</version>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>1.5.16</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.5.16</version>
+            <scope>test</scope>
```

Comment:
Has there been locally tested? 
I worry that having these versions of logback will break the structured logging messages.